package.json

```diff
@@ -52,12 +52,15 @@
     "@storybook/test": "^8.4.2",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
+    "@typescript-eslint/eslint-plugin": "^8.16.0",
```

Comment:
Are we using these 3 new plugins ?

Reply:
i was playing around with the. configs, forgot to remove it. thanks for the reminder